playbooks/development-environments.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Development environments
+description: Getting set up to work on new projects
+---
+
+# Development environments
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Goals](#goals)
+- [Prerequisites](#prerequisites)
+- [Project set-up](#project-set-up)
+- [Shared configuration](#shared-configuration)
+- [Examples](#examples)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Goals
+
+- New working environments are set up quickly, easily, and consistently (ideally with a single command).
+- Team- or project-wide configuration changes are rolled out without needing a lot of communication and
+  coordination.
+- Projects adopting favored technologies leverage common, simple set-up.
+- Developers using common tools (mac, bash, homebrew, nvm...) get productive without any special set-up.
+- Developers choosing less common tools are still productive, if necessary using the basic set-up as a guide.
+- Projects default to configuration values that are appropriate for development (just as they run in "development"
+  mode by default).
+- Both local and containerized development are supported, with minimal duplication.
+- Sensitive values are never committed to repositories, whether open- or closed-source.
+- Other things being equal, less configuration, simpler-setup, and fewer dependencies are better than more.
+
+## Prerequisites
+
+Setup tooling is optimized for a default set of tools:
+
+- Mac computers
```

Comment:
We don't _need_ to call this out, but I'm curious if we have a current recommendation for computer specs for devs (i.e. 16 vs. 32GB of RAM)?